File: src/1005.c
/*
 * 1005 - Average 1
 * Author: Otavio
 * Date: 25/04/2024
 */
#include <stdio.h>
 
int
main()
{
    double a, b;
    
    scanf("%lf", &a);
    scanf("%lf", &b);
    
    double avg = (a*3.5 + b*7.5)/11.0;
    
    printf("MEDIA = %.5lf\n", avg);
    return 0;
}

File: src/JS/1005.js
/*
 * 1005 - Average 1
 * Author: Otavio
 * Date: 31/08/2024
 */
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');

const A = parseFloat(lines.shift());
const B = parseFloat(lines.shift());

const MEDIA = ((A*0.35) + (B*0.75))/1.1;
console.log(`MEDIA = ${MEDIA.toFixed(5)}`);
